components/Icon/Icon.unit.test.jsx

```diff
@@ -19,6 +19,12 @@ describe('<Icon />', () => {
     const withLargeSize = renderer
       .create(<Icon name="search" size="large" />)
       .toJSON();
+    const withXSmallSize = renderer
+      .create(<Icon name="search" size="xlarge" />)
```

Comment:
I think the size for this was suposed to be "xsmall" and not "xlarge"? What do you think?

Reply:
My bad!